bin/tasks/gitInfo.js

```diff
@@ -21,8 +21,10 @@ const TesterSkipBuildMutation = `
 export const setGitInfo = async (ctx, task) => {
   const { branchName, patchBaseRef, fromCI: ci } = ctx.options;
   ctx.git = await getCommitAndBranch({ branchName, patchBaseRef, ci, log: ctx.log });
-  ctx.git.slug = ctx.git.slug || (await getSlug());
   ctx.git.version = await getVersion();
+  if (!ctx.git.slug) {
+    ctx.git.slug = await getSlug().catch((e) => ctx.log.warn('Failed to retrieve slug', e));
+  }
```

Comment:
What is this slug, and why is it optional? What do we miss if we don't have it?

Reply:
@ghengeveld why not `try { } catch {}`?

I think it's confusing to use `await` and `.catch()`

Reply:
The slug is something like `chromaui/chromatic-cli` and is needed to determine whether we're dealing with a forked repository or not, which we need to know for cross-fork PR support.

I used `.catch` because it's way terser than try/catch and given this is such a benign thing I don't think it deserves to be so verbose.